src/MB_Toolbox.php

```diff
@@ -201,6 +201,97 @@ public function createDrupalUser($user) {
   }
 
   /**
+   *
+   */
+  public function createNorthstarUser($user) {
+
+    $northstarAPIConfig = $this->mbConfig->getProperty('northstar_config');
+    if (empty($northstarAPIConfig['host'])) {
+      throw new Exception('MB_Toolbox->createNorthstarUser() northstar_config missing host setting.');
+    }
+
+    // Required - at least one of email or mobile must be set.
+     $requiredSet = false;
+    if (isset($user->email)) {
```

Comment:
`isset()` just ensures the `email` property exists and not `null`. `isset()` will return true even if it equals `false` or an empty string.

`!empty()` does the same plus check the value to be truthy, which exactly what we need here.